fidimag/common/nebm_tools.py

```diff
@@ -0,0 +1,193 @@
+from __future__ import print_function
+from __future__ import division
+import numpy as np
+
+
+def cartesian2spherical(y_cartesian):
```

Comment:
Maybe add a comment just to point out which definition of spherical coords you've used, i.e what are the bounds of theta and phi